### DISCUSSION TITLE
Tricaster 410 AE3

### DISCUSSION THREAD

**Discussion by Mo96M** [josephdadams/TallyArbiter#460] [Q&A]
Hi, my company's Tricaster410 has a physical tally output but it's not working correctly. So I've tried to use TallyArbiter because i know that NDI Tally is working with the Studio Monitor. In TallyArbiter i also receive tally data from the Tricaster, as you can see attached. But i don't get any signal on my listeners. I've tested it with the tally view page in Firefox.

I'm not 100% sure if everything is setup correctly.
Does anyone has an idea? 

Kind regards 
Moritz
 
![TC410 tally data TallyArbiter](https://user-images.githubusercontent.com/62653899/176703099-35f61adc-5fa4-4aad-9ae6-2d66f85b1461.PNG)

**Comment by JTF4** [comment]
What version are you currently running?

**Reply by Mo96M** [reply]
2.0.16

**Comment by Mo96M** [comment]
Maybe it's just a problem with the Source naming. When I'm selecting the sources for the input settings on the tricaster it's Source: Local -> Input 1 (or 2, 3, 4). But I can rename them for a better overview, so they are labeled KAM 1 - 4.

For NDI sources it's the name of the device the source is coming from and then the source name.

**Comment by josephdadams** [comment]
Could this be related to #24 ?

**Reply by JTF4** [reply]
This would be a very plausible thing.

**Comment by JTF4** [comment]
@Mo96M can you upload your config file so I can have a look at it?

**Reply by Mo96M** [reply]
Hi David,

you can find my config file attached.

[config.json.txt](https://github.com/josephdadams/TallyArbiter/files/9038645/config.json.txt)

thanks for your effort

**Reply by JTF4** [reply]
Can you try running the V3 beta and seeing if the issue persists? I don't recommend using V3 in production yet as we are still working on it, but if it works, then it gives me a good place to start and I'll see if I can do a V2 update to fix your issue.

**Reply by Mo96M** [reply]
Now I have a dropdown list with sources, but it's seem like there are some formatting mistakes?

![TA3 0 4 Sources](https://user-images.githubusercontent.com/62653899/178226110-970a808f-6464-4cee-94d1-124bc11bae69.png)

The tricaster is connected but i don't get any tally data.


```
{
  "security": {
    "jwt_private_key": "KNe0avpKv7KyKKiCtK8r5Aq+dE5X/1E9/atRGttRBM9lhd4G8KtNH4aM8p18fTEszKERhNxYx0HIiKepti530ufQCAA9ZKqpzX9t8E7OABeW0P6CffTqLnva4SYhrbMQYkj5MIxNM1+3ziT6zE5WnpEWpKKM4ymHTuzpG2Pvp2G+DFHnMx5ebUrFvCWC1HfCJemnbzcXGFt7ehCWuhUqkBbsqHlV8Lt7cPvhEroxPVrN+/yruE7qO+pw9WQ5nxOXkUCTKYA/wEWPtQO3WVjoy9lnssObhVBpB177Zu8b/6vYR3xiCJu1YgW0anMy9IuCBgV9eXL6rC1qSP7IwlBmOg=="
  },
  "users": [
    {
      "username": "admin",
      "password": "$2b$10$F9IgpSo/p/VkKjqRtMKa8./Cz99J272tGM3Iau5sbBRPR7UPA.J52",
      "roles": "admin"
    },
    {
      "username": "producer",
      "password": "$2b$10$QavCSOZSzGpNFp7Z6tY5Denvr/TKiLdo9d9pB5U6FHrh4nls3ZApa",
      "roles": "producer"
    }
  ],
  "cloud_destinations": [],
  "cloud_keys": [],
  "device_actions": [],
  "device_sources": [
    {
      "deviceId": "40249c6d",
      "sourceIdx": "0",
      "address": "p",
      "rename": true,
      "sourceId": "46a870f6",
      "id": "5b4bbc9e"
    },
    {
      "deviceId": "7caad215",
      "sourceIdx": "0",
      "address": "r",
      "rename": true,
      "sourceId": "46a870f6",
      "id": "e279cea9"
    },
    {
      "deviceId": "fa935a4f",
      "sourceIdx": "0",
      "address": "o",
      "rename": true,
      "sourceId": "46a870f6",
      "id": "a680047b"
    },
    {
      "deviceId": "ef62c1d9",
      "sourceIdx": "0",
      "address": "g",
      "rename": true,
      "sourceId": "46a870f6",
      "id": "5a1de8aa"
    }
  ],
  "devices": [
    {
      "name": "KAM 1",
      "description": "Kamera 1",
      "enabled": true,
      "id": "40249c6d"
    },
    {
      "name": "KAM 2",
      "description": "Kamera 2",
      "enabled": true,
      "id": "7caad215"
    },
    {
      "name": "KAM 3",
      "description": "Kamera 3",
      "enabled": true,
      "id": "fa935a4f"
    },
    {
      "name": "KAM 4",
      "description": "Kamera 4",
      "enabled": true,
      "id": "ef62c1d9"
    }
  ],
  "sources": [
    {
      "data": {
        "ip": "192.168.137.13"
      },
      "name": "TC410",
      "sourceTypeId": "f2b7dc72",
      "reconnect": true,
      "enabled": true,
      "id": "46a870f6",
      "max_reconnects": 5,
      "connected": true
    }
  ],
  "tsl_clients": [],
  "tsl_clients_1secupdate": false,
  "bus_options": [
    {
      "id": "e393251c",
      "label": "Preview",
      "type": "preview",
      "color": "#3fe481",
      "priority": 50
    },
    {
      "id": "334e4eda",
      "label": "Program",
      "type": "program",
      "color": "#e43f5a",
      "priority": 200
    },
    {
      "id": "12c8d699",
      "label": "Aux 1",
      "type": "aux",
      "color": "#0000FF",
      "priority": 100
    },
    {
      "id": "12c8d689",
      "label": "Aux 2",
      "type": "aux",
      "color": "#0000FF",
      "priority": 100
    }
  ],
  "externalAddress": "http://0.0.0.0:4455/#/tally",
  "remoteErrorReporting": false,
  "uuid": "facdc1cb"
}
```

**Reply by Mo96M** [reply]
There are also some config errors. I've added the red marked line based on an error comment like the one about the missing "reconnect_interval"

![TA3 0 4 Config](https://user-images.githubusercontent.com/62653899/178279003-cfde5c20-9b49-42da-9dd8-e84db17b6b91.png)
